coalib/results/Diff.py

```diff
@@ -413,7 +430,7 @@ def __add__(self, other):
             if change.add_after is not False:
                 result.add_lines(line_nr, change.add_after)
             if change.change is not False:
-                result.change_line(line_nr, change.change[0], change.change[1])
+                result.modify_line(line_nr, change.change[1])
```

Comment:
Should I move all my modifications, that replace `change_line` with `modify_line` into the new commit, or just in functions that i have not altered in other ways?